easyrsa3/easyrsa

```diff
@@ -860,6 +862,8 @@ at: $crt_in"
 
 	"$EASYRSA_OPENSSL" ca -utf8 -revoke "$crt_in" -config "$EASYRSA_SAFE_CONF" || die "\
 Failed to revoke certificate: revocation command failed."
+	rm -f "$crt_in"
+	! [ -e "$key_in" ] || rm -f "$key_in"
```

Comment:
This syntax is wonky.  What are you trying to do with that exclamation point?

Reply:
I normally avoid returning false (useful to "set -e").

[ -e "$key_in" ] && rm -f "$key_in"

Will be false if file is absent. I can change to a "if ... " or '&&' if needed. Which one do you prefer?